python/lsst/meas/base/forcedPhotCcd.py

```diff
@@ -188,11 +191,11 @@ def makeIdFactory(self, dataRef):
                              (e.g. visit, raft, sensor for LSST data).
         """
         expBits = dataRef.get("ccdExposureId_bits")
-        expId = long(dataRef.get("ccdExposureId"))
+        expId = int(dataRef.get("ccdExposureId"))
```

Comment:
Is it ok for this to not be `long` on python 2? Noting that on 64-bit python "int" is 64-bit.